bebop_controller.py

```
@@ -0,0 +1,644 @@
+    """FL Studio에서 스크립트가 로드될 때 호출"""
+    print("비밥 솔로 & 코드 컨트롤러 초기화됨")
+    
+    # 설정 로드
+    load_settings()
+    
+    # 채널 설정
+    try:
+        channels.setChannelName(MELODY_CHANNEL, "비밥 솔로")
+        channels.setChannelName(CHORD_CHANNEL, "코드 컴핑")
+        print(f"채널 설정 완료: 채널 {MELODY_CHANNEL}=비밥 솔로, 채널 {CHORD_CHANNEL}=코드 컴핑")
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)